FE/src/templates/AddIssueWrite/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Badge } from '@/components/common/Badge';
+import * as S from './style';
+import { Text } from '@/components/common/Text';
+
+export function AddIssueWrite() {
+  return (
+    <S.AddIssueWrite>
+      <Badge
+        imgSRC="https://ca.slack-edge.com/T74H5245A-U02RA7K501L-3c55992367ca-512"
```

Comment:
여기 상수로 관리해서 빼놓으면 좋을 것 같네요 
```suggestion
        imgSRC="{profileImage}"
```